django/template/loaders/base.py

```diff
@@ -44,3 +93,10 @@ def reset(self):
         templates or cached loader modules).
         """
         pass
+
+    @property
+    def supports_recursion(self):
+        """
+        Used during deprecation period of non-recursive loaders.
```

Comment:
Nevermind, I see what you mean now about the ``supports_recursion`` deprecation. I'll update it.